src/components/App/App.tsx

```diff
@@ -1,13 +1,51 @@
-import React from 'react';
-import logo from '../src/assets/logo.svg';
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
 import './App.css';
+import mapIcon from '../../assets/WorldMap.jpg';
+import Map from '../Map/Map';
+import apiCalls from '../../api/apiCalls';
+
+interface IState{
```

Comment:
We can add to this IState interface as we determine all the stuff we'll be passing to Map and QuizPage!